src/python_testing/TC_ACL_2_4.py

```diff
@@ -0,0 +1,1628 @@
+            TestStep(13, "TH1 reads DUT Endpoint 0 AccessControl cluster ACL attribute",
+                     "Result is SUCCESS, value is list of AccessControlEntryStruct containing 3 elements with null targets"),
+            TestStep(14, "TH1 writes DUT Endpoint 0 AccessControl cluster ACL attribute with 2 elements", "Result is SUCCESS"),
+            TestStep(15, "TH1 reads DUT Endpoint 0 AccessControl cluster ACL attribute",
+                     "Result is SUCCESS, value is list of AccessControlEntryStruct containing 2 elements"),
+            TestStep(16, "TH1 writes DUT Endpoint 0 AccessControl cluster ACL attribute with ProxyView privilege", "Result is SUCCESS"),
+            TestStep(17, "TH1 reads DUT Endpoint 0 AccessControl cluster ACL attribute",
+                     "Result is SUCCESS, value is list of AccessControlEntryStruct containing 2 elements with ProxyView privilege for the last element"),
+            TestStep(18, "TH1 reads DUT Endpoint 0 AccessControl cluster SubjectsPerAccessControlEntry attribute",
+                     "Result is SUCCESS, value is stored as MAXSUBJECTS"),
+            TestStep(19, "TH1 writes DUT Endpoint 0 AccessControl cluster ACL attribute with max subjects", "Result is SUCCESS"),
+            TestStep(20, "TH1 reads DUT Endpoint 0 AccessControl cluster ACL attribute",
+                     "Result is SUCCESS, value contains MAXSUBJECTS random node IDs"),
+            TestStep(21, "TH1 writes DUT Endpoint 0 AccessControl cluster ACL attribute with 4 valid CATs as subjects in the last entry", "Result is SUCCESS"),
+            TestStep(22, "TH1 reads DUT Endpoint 0 AccessControl cluster ACL attribute",
+                     "Result is SUCCESS, value is list of AccessControlEntryStruct containing 2 elements with last element having 4 valid CATs as subjects"),
+            TestStep(23, "TH1 reads DUT Endpoint 0 AccessControl cluster TargetsPerAccessControlEntry attribute",
+                     "Result is SUCCESS, value is stored as MAXTARGETS"),
+            TestStep(24, "TH1 writes DUT Endpoint 0 AccessControl cluster ACL attribute with max targets", "Result is SUCCESS"),
+            TestStep(25, "TH1 reads DUT Endpoint 0 AccessControl cluster ACL attribute",
+                     "Result is SUCCESS, value contains MAXTARGETS targets"),
+            TestStep(26, "TH1 reads DUT Endpoint 0 AccessControl cluster AccessControlEntriesPerFabric attribute",
+                     "Result is SUCCESS, value is stored as MAXENTRIES"),
+            TestStep(27, "TH1 writes DUT Endpoint 0 AccessControl cluster ACL attribute with a list of MAXENTRIES elements", "Result is SUCCESS"),
+            TestStep(28, "TH1 reads DUT Endpoint 0 AccessControl cluster ACL attribute",
+                     "Result is SUCCESS, value is list of AccessControlEntryStruct containing MAXENTRIES elements"),
+            TestStep(29, "TH1 writes DUT Endpoint 0 AccessControl cluster ACL attribute testing PASE auth mode",
```

Comment:
I have added a step 29 in test plan which involves resetting the ACLs, can you please add it, and move all below steps by one  count. Thanks!
```suggestion
            TestStep(30, "TH1 writes DUT Endpoint 0 AccessControl cluster ACL attribute testing PASE auth mode",
```

Reply:
Thank you, I have updated to match the new test step information for test steps 29 and 30 to match with the test plan.

Reply:
Thanks, just a small addition
```suggestion
            TestStep(29, "TH1 writes DUT Endpoint 0 AccessControl cluster ACL attribute with acl_original",
```